pasting the full compiler invocation for every cu is redundant, because failures
already print the compiler invocation (where it matters), and the rest is spam
--
--- a/src/core/gn_run.pro
+++ b/src/core/gn_run.pro
@@ -45,7 +45,6 @@
     isEmpty(enableThreads):macos {
         gn_threads = "--threads=1"
     }
-    isEmpty(ninjaflags):!silent: ninjaflags = "-v"
     build_pass:build_all: default_target.target = all
     else: default_target.target = first
     default_target.depends = runninja
